src/@types/Calendar.tsx

```diff
@@ -0,0 +1,2 @@
+export const MONTH_NAMES_SHORT = [`Jan`, `Feb`, `Mar`, `Apr`, `May`, `Jun`, `Jul`, `Aug`, `Sep`, `Oct`, `Nov`, `Dec`]
+export const WEEKDAY_NAMES_SHORT = [`Sun`, `Mon`, `Tue`, `Wed`, `Thu`, `Fri`, `Sat`]
```

Comment:
Need to get it automatically for current locale, else need to localize it

Reply:
got it, found a way to generate them using date-fns